src/lib/sass.worker.js

```diff
@@ -1,3 +1,15 @@
+// Temporary fix from: https://github.com/medialize/sass.js/issues/96#issuecomment-424386171
```

Comment:
Definitely don't like touching 3rd party files but it's your call, ultimately.

Reply:
Oh, OK, if this is going to be fixed upstream.

Reply:
yep here: https://github.com/medialize/sass.js/issues/96#issuecomment-425373458 👍